gadgets/src/range.rs

```diff
@@ -0,0 +1,153 @@
+//! Specialized version to replace LtChip for u32/u64/u128 check
+//!
+//! only constrain input expressions to be in given range
+//! do not output the result
+//!
+//! u8 check can either use U8Table or group every 2 values and lookup U16Table
```

Comment:
This comment is incorrect (remove "or group every 2 values and lookup U16Table").